src/GraphQL.AotCompilationSample/GraphQL.AotCompilationSample.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
I would like to unify TFMs - either `netX.0` everywhere or just `netX`.

Reply:
I think most are correct but I might have missed one or two.  I believe it should be `netX.0` as this is the MS recommendation and works better for tests in Visual Studio